package.json

```diff
@@ -354,6 +354,236 @@
 						}
 					}
 				}
+			},
+			{
+				"type": "reactnativedirect",
+				"label": "React Native Direct",
+				"program": "./src/debugger/direct/reactNativeDirectDebugEntryPoint.js",
+				"runtime": "node",
+				"enableBreakpointsFor": {
+					"languageIds": [
+						"javascript",
+						"typescript",
+						"javascriptreact",
+						"typescriptreact"
+					]
+				},
+				"configurationSnippets": [
+					{
+						"label": "React Native (Hermes): Debug Android",
+						"description": "%reactNative.snippets.debugAndroid.description%",
+						"body": {
+							"name": "Debug Android (Hermes)",
+							"cwd": "^\"\\${workspaceFolder}\"",
+							"type": "reactnativedirect",
+							"request": "launch",
+							"platform": "android"
+						}
+					},
+					{
+						"label": "React Native (Hermes): Attach to packager",
+						"description": "%reactNative.snippets.attachPackager.description%",
+						"body": {
+							"name": "Attach to packager (Hermes)",
+							"cwd": "^\"\\${workspaceFolder}\"",
+							"type": "reactnativedirect",
+							"request": "attach"
+						}
+					}
+				],
+				"configurationAttributes": {
+					"attach": {
+						"required": [
+							"cwd"
+						],
+						"properties": {
+							"program": {
```

Comment:
Please, remove all unverified properties. We don't have yet information about how we support them for teh new debugger, so let's add them later as more information will be available.